cmd/osbuild-composer/main.go

```diff
@@ -80,10 +80,12 @@ func main() {
 	}
 
 	logrus.Info("Loaded configuration:")
-	err = DumpConfig(*config, logrus.StandardLogger().WriterLevel(logrus.InfoLevel))
+	dumpWriter := logrus.StandardLogger().WriterLevel(logrus.DebugLevel)
+	err = DumpConfig(*config, dumpWriter)
```

Comment:
I guess it does not matter, but would `defer dumpWiter.Close()` make sense here (and in the other places where close is used)? I.e. similar to how there is always a "defer f.Close()" for other resources?

Reply:
I want to close and unblock the goroutine as soon as possible, there is no reason to hold it off until program exits. It is either this, or I can create an anonymous function with `defer` which is IMHO an overkill.